doc/admin/install-kubernetes.md

```diff
@@ -30,30 +48,58 @@ All the following files are located under `extras/kubernetes`.
 $ kubectl create -f glusterfs-daemonset.json
 ```
 
-* Get node name by running:
+* Get node names by running:
 
 ```
 $ kubectl get nodes
 ```
 
-* Deploy gluster container onto specified node by setting the label `storagenode=glusterfs` on that node
+* Deploy gluster container onto a specified node by setting the
+  label `storagenode=glusterfs` on that node.
 
 ```
 $ kubectl label node <...node...> storagenode=glusterfs
 ```
 
-Repeat as needed.  Verify that the pods are running on the nodes.
+Repeat as needed. Verify that the pods are running on the nodes, at least
+three pods should be running.
 
 ```
 $ kubectl get pods
 ```
 
-* Next we need to deploy the Pod and Service Heketi Service Interface to the GlusterFS cluster. In the repo you cloned, there will be a deploy-heketi-deployment.json file. 
+* Next we will create a service account for Heketi:
+
+```
+$ kubectl create -f heketi-service-account.json
+```
+
+* We must now establish the ability for that service account to control
+  the gluster pods. We do this by creating a cluster role binding for
+  our newly created service account.
+
+```
+$ kubectl create clusterrolebinding heketi-gluster-admin --clusterrole=edit --serviceaccount=default:heketi-service-account
+```
+
+* Now we need to create a Kubernetes secret that will hold the configuration
+  of our Heketi instance. The configuration file must be set to use the
+  `kubernetes` executor in order for the Heketi server to control the
+  gluster pods. Beyond that, feel free to experiment with the configuration
+  options.
+
+```
+$ kubectl create secret generic heketi-config-secret --from-file=./heketi.json
+```
+
+* Next we need to deploy the Pod and Service Heketi Service Interface to
```

Comment:
This sentence doesn't make much sense. Reword. I THINK you want to say something about deploying the deploy-heketi Pod and Service?

Reply:
Hah. I can claim that's not my word salad as its in the original even though I touched the line. However, I can't say how I managed to miss when I was doing my other wording clean-ups. Oh well. :-)